api/v1alpha1/etcd.go

```diff
@@ -194,6 +194,10 @@ type EtcdConfig struct {
 	// Quota defines the etcd DB quota.
 	// +optional
 	Quota *resource.Quantity `json:"quota,omitempty"`
+	// SnapshotCount defines the number of applied Raft entries to hold in-memory before compaction.
```

Comment:
have you run into a case where you needed this as configurable? Can you please elaborate that case?

Reply:
I've never run into that personally but this PR solves the issue [717](https://github.com/gardener/etcd-druid/issues/717) where the second point is to make `snapshot-count` configurable, so went ahead with it as that made sense. I can remove this if it's not good to have.

Reply:
ok then lets keep it.